engines/supernova/game-manager.cpp

```diff
@@ -360,6 +362,8 @@ void GameManager::processInput() {
 		onInventoryArrowDown
 	} mouseLocation;
 
+	_ttsMan = g_system->getTextToSpeechManager();
```

Comment:
If you are going to make this a member variable for the `GameManager` class, don't set it every time  `processInput()` gets called. Instead it should be set in the `GameManager` constructor.

But I don't see the benefit of having it as a member variable here.